spec/serializers/api/v2/green_lanes/categorisation_serializer_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Api::V2::GreenLanes::CategorisationSerializer do
+  subject(:serialized) do
+    described_class.new(::GreenLanes::Categorisation.load_from_string(json_string)).serializable_hash.as_json
+  end
+
+  let(:json_string) do
+    '[{
+          "category": "1",
+          "regulation_id": "D0000001",
+          "measure_type_id": "400",
+          "geographical_area": "1000",
+          "document_codes": [],
+          "additional_codes": []
+        }]'
```

Comment:
the indentation looks strange here

Reply:
I safer approach is to use Ruby structure and convert it `to_json` :
```
[{
  "category"=>"1", 
  "regulation_id"=>"D0000001", 
  "measure_type_id"=>"400", 
  "geographical_area"=>"1000", 
  "document_codes"=>[], 
  "additional_codes"=>[]
}].to_json
```
This way the object is parsed and the json format is correct.